### PR TITLE
docs: fixed broken links

### PR DESCRIPTION
**Pull request by vinckr**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: docs/docs/concepts/credentials/username-email-password.mdx [hunk 1]
```diff
@@ -288,7 +288,7 @@ Assuming your traits schema is as follows:
 
 And an identity registers with the following JSON payload (more on registration
 in
-[Selfservice Registration](../self-service/flows/user-login-user-registration.md)):
+[Selfservice Registration](../self-service/flows/user-login-user-registration)):
```
Comment (aeneasr) [inline docs/docs/concepts/credentials/username-email-password.mdx:291]: Point one more up :)

**Comment by aeneasr** [comment]
@vinckr can you please finish this?

File: docs/docs/concepts/credentials/username-email-password.mdx (hunk 1 above)
Comment (vinckr) [reply docs/docs/concepts/credentials/username-email-password.mdx:291]: What do you mean by point up?

Comment (vinckr) [reply docs/docs/concepts/credentials/username-email-password.mdx:291]: What do you mean by point up?

Comment (jfcurran) [inline docs/docs/concepts/credentials/username-email-password.mdx:291]: [Selfservice Registration](../../self-service/flows/user-login-user-registration)):

Comment (aeneasr) [reply docs/docs/concepts/credentials/username-email-password.mdx:291]:
```suggestion
[Selfservice Registration](../../self-service/flows/user-login-user-registration)):
```

**Pull request merged**